src/thing/get.test.ts

```diff
@@ -871,6 +990,136 @@ describe("getDatetimeAll", () => {
   });
 });
 
+describe("getDateAll", () => {
+  it("returns the date values for the given Predicate", () => {
+    const thingWithDates = getMockThingWithLiteralsFor(
+      "https://some.vocab/predicate",
+      "1955-06-08",
+      "1990-11-12",
+      "date"
+    );
+    const expectedDate1 = new Date(Date.UTC(1955, 5, 8));
+    const expectedDate2 = new Date(Date.UTC(1990, 10, 12));
+
+    expect(
+      getDateAll(thingWithDates, "https://some.vocab/predicate")
+    ).toEqual([expectedDate1, expectedDate2]);
+  });
+
+  it("accepts Properties as Named Nodes", () => {
+    const thingWithDates = getMockThingWithLiteralsFor(
+      "https://some.vocab/predicate",
+      "1955-06-08",
+      "1990-11-12",
+      "date"
+    );
+    const expectedDate1 = new Date(Date.UTC(1955, 5, 8));
+    const expectedDate2 = new Date(Date.UTC(1990, 10, 12));
+
+    expect(
+      getDateAll(
+        thingWithDates,
+        DataFactory.namedNode("https://some.vocab/predicate")
+      )
+    ).toEqual([expectedDate1, expectedDate2]);
+  });
+
+  it("returns an empty array if no datetime values were found", () => {
```

Comment:
There's a couple of other mentions of `datetime`s below too :)

```suggestion
  it("returns an empty array if no date values were found", () => {
```